### ISSUE TITLE
Bug(sort-imports): Endless newlines added between imports when semicolons are used

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [azat-io/eslint-plugin-perfectionist#12]
### Describe the bug

When running ESLint with `--fix` on a file containing imports that end with `;` and the `perfectionist/sort-imports` rule believes should be separated, a newline is always added before the first import's `;`. This causes the fixer to continuously add more newlines before the `;`.

### Code example

Before:

```js
import abc from "abc";
import def from "./def";
```

After:

```js
import abc from "abc"









;
import def from "./def";
```

### ESLint version

8.42.0

### ESLint Plugin Perfectionist version

1.1.0

### Additional comments

I'm using a config that just extends the recommended defaults:

```js
module.exports = {
  extends: ["plugin:perfectionist/recommended-alphabetical"],
  parserOptions: {
    ecmaVersion: 2020,
    sourceType: "module",
  },
  plugins: ["perfectionist"],
};
```

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Thank you for the issue! I was able to reproduce the problem. I will get to it soon.

**Comment by azat-io** [comment]
Fixed in e435f91094cd1758d4c095f2a8b66f9672b6ecb0

Will be released today

**Comment by azat-io** [comment]
Released in [v1.1.1](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v1.1.1)

**Comment by JoshuaKGoldberg** [comment]
Amazing, thanks @azat-io! Very speedy ⚡
